tests/task_comments/test_api_create_task_comment.py

```diff
@@ -76,6 +78,20 @@ def test_task_comment_creation_api_with_relation_not_existing(self):
         self.assertEqual(HTTPStatus.NOT_FOUND.value, actual_response.status_code)
         self.assertDictEqual(expected_response, json.loads(actual_response.data))
 
+    def test_task_comment_creation_api_with_unaccepted_relation(self):
+        auth_header = get_test_request_header(self.fourth_user.id)
+        expected_response = messages.UNACCEPTED_STATE_RELATION
+        actual_response = self.client.post(
+            f"mentorship_relation/{self.relation_id_one}/task/{self.task_id_one}/comment",
+            follow_redirects=True,
+            headers=auth_header,
+            content_type="application/json",
+            data=json.dumps(dict(comment="comment")),
+        )
+
+        self.assertEqual(HTTPStatus.BAD_REQUEST.value, actual_response.status_code)
```

Comment:
```suggestion
        self.assertEqual(HTTPStatus.BAD_REQUEST, actual_response.status_code)
```

I think you can remove the `.value` part and it will work anyways :)